### DISCUSSION TITLE
nginx config suggestion

### DISCUSSION THREAD

**Discussion by mdougher** [redimp/otterwiki#114] [General]
Just sharing what i found that might help save some others some time.

i used the config at https://otterwiki.com/Installation#reverse-proxy for configuring nginx for my install.
I utilize a lot of PDF attachments, and pretty quickly hit the default limit on upload size for nginx. 
Might be a good idea to add a note in the docs to consider changing:
```
location / {
    client_max_body_size 100M;
}
```
with whatever appropriate size is needed for your setup.

**Comment by redimp** [comment]
Thanks for sharing this, I've added it to the Installation documentation.
